tests/integration/session/test_multiple_login.py

```diff
@@ -13,7 +13,7 @@ def setUp(self):
 
         self.cache = {}
 
-    def launchSurvey(self, client, schema_name="test_textfield", **payload_kwargs):
+    def launchSurvey(self, schema_name, client, **payload_kwargs):
```

Comment:
Any reason this was changed?. This is inconsistent with post where it passes a client as the first arg. My preference is to keep as it was.

Reply:
I forgot why, but changes have been reverted now 👍